packages/big-design/src/components/Panel/Contents/styled.tsx

```diff
@@ -0,0 +1,66 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { Box } from '../../Box';
+
+import { PanelContentProps } from './Contents';
+
+export const StyledPanelContentsWrapper = styled(Box)<Omit<PanelContentProps, 'children'>>`
+  ${({ theme, scrollable }) =>
+    scrollable &&
+    css`
+      background-image: linear-gradient(
+          0deg,
+          ${theme.colors.secondary40} 0%,
+          transparent 1px,
+          transparent 100%
+        ),
+        linear-gradient(180deg, ${theme.colors.secondary40} 0%, transparent 1px, transparent 100%);
+      overflow: auto;
+    `}
+
+  ${({ height }) =>
+    height &&
+    css`
+      height: ${height};
+    `}
+
+  ${({ theme, padded }) =>
+    padded !== true &&
+    css`
+      margin-inline: -${theme.spacing.medium};
+    `}
+
+  ${({ theme }) => theme.breakpoints.tablet} {
+    ${({ theme, padded }) =>
+      padded !== true &&
+      css`
+        margin-inline: -${theme.spacing.xLarge};
```

Comment:
these minus margins are a bit awkward

both the technique, plus they're trying to undo the Panel's padding, without referencing the Panel's padding value
so the values could get out of sync

I'm also guessing that we wouldn't want `PanelContents` ever being used outside of a `Panel`
So I'm wondering why not hide it as an implementation detail, rather than expose it as its own component

e.g.

```tsx
<Panel
  header="Panel header"
  fullWidthContents
>
  <Text>
    Hello there
  </Text>
</Panel>
```

or with a `contents` namespace to absorb the other new settings (scroll behaviour etc)

```tsx
<Panel
  header="Panel header"
  contents={{ fullWidth: true }}
>
  <Text>
    Hello there
  </Text>
</Panel>
```